shared_stack.bash

```diff
@@ -133,7 +136,10 @@ for tag in $(comm -1 -3 <(sort -u tags.deleted) $tmp/tags); do
        echo "conda activate $LSST_CONDA_ENV_NAME" > switch.sh
        echo "$LSST_CONDA_ENV_NAME" > env_name
      fi
-  ) && [ ! -d "$dir" ] && $dryrun mv "${dir}.$tmp" "$dir"
+  )
+  [ "$?" -eq 0 ] && [ ! -d "$dir" ] && $dryrun mv "${dir}.$tmp" "$dir"
```

Comment:
It's fine for the subshell to fail?

Reply:
Yes, if it fails it will be cleaned up by the next bit of code, and subsequent runs should retry.